### ISSUE TITLE
Add Gnosis Safe connector

### ISSUE THREAD

**Issue by NoahZinsmeister** [NoahZinsmeister/web3-react#349]
it'd be great to have an official gnosis safe connector

**Comment by mattcasey** [comment]
Hi, I am interested in getting this to work and can contribute. I am still using v6 though, because I'm not sure if v8 is ready for 'prime time'

**Comment by NoahZinsmeister** [comment]
closed by 3becf08785e658df5bd9121b582172eed3f90a0c

**Comment by Christopher-Stevers** [comment]
Thanks for setting this up. Do you know of any examples of this working in the wild. I'm 
I'm trying to connect with gnosis, and I'm getting `TypeError: connector.on is not a function`.
The stack trace from next gives me this
```useEffect(function () {
  491 |     if (connector) {
> 492 |       connector.on(ConnectorEvent.Update, handleUpdate).on(ConnectorEvent.Error, handleError).on(ConnectorEvent.Deactivate, handleDeactivate);
  493 |     }
  494 | 
  495 |     return function () {```
  
  Any ideas what might be wrong here? I'm passing the same props to the GnosisSafe connector as I am to the WalletConnectConnector (which is working).
